test/service/test_async_manifest_service.py

```diff
@@ -53,6 +56,8 @@ def setUpModule():
 
 
 patch_step_function_helper = mock.patch('azul.service.async_manifest_service.AsyncManifestService.step_function_helper')
+service = load_app_module('service')
+state_machine_name = config.state_machine_name(service.manifest_service_name)
```

Comment:
```suggestion
state_machine_name = 'foo'
```

Since it's a unit test we should be able to use a mock value.